blosc2/schunk.py

```diff
@@ -929,7 +987,9 @@ def __dealloc__(self):
         super().__dealloc__()
 
 
-def open(urlpath, mode="a", offset=0, **kwargs):
+@_inherit_doc_parameter(SChunk.__init__, "mmap_mode:", {r"\* - 'w\+'[^*]+": ""})
+@_inherit_doc_parameter(SChunk.__init__, "initial_mapping_size:", {r"r\+ w\+, or c": "r+ or c"})
+def open(urlpath, mode="a", mmap_mode=None, initial_mapping_size=None, offset=0, **kwargs):
```

Comment:
The positions of the mmap-related params are actually breaking the API. I'd move them into **kwargs and unpack them internally.  Using the `kwargs` mechanism allows for more robust API (e.g. future I/O plugins).